dotnet/samples/Demos/Hosting/Agent/ChatWithAgent.ApiService/Controllers/AgentCompletionsController.cs

```diff
@@ -15,14 +20,17 @@ namespace ChatWithAgent.ApiService;
 public sealed class AgentCompletionsController : ControllerBase
 {
     private readonly ILogger<AgentCompletionsController> _logger;
+    private readonly ChatCompletionAgent _agent;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="AgentCompletionsController"/> class.
     /// </summary>
     /// <param name="logger">The logger.</param>
-    public AgentCompletionsController(ILogger<AgentCompletionsController> logger)
+    /// <param name="agent">The agent.</param>
+    public AgentCompletionsController(ILogger<AgentCompletionsController> logger, ChatCompletionAgent agent)
```

Comment:
Move logger to the last parameter, is a good practice, normally is less important.

```suggestion
    public AgentCompletionsController(ChatCompletionAgent agent, ILogger<AgentCompletionsController> logger)
```